subprojects/client-protocol/src/main/java/org/gradle/profiler/client/protocol/Client.java

```diff
@@ -1,19 +1,26 @@
 package org.gradle.profiler.client.protocol;
 
+import org.gradle.profiler.client.protocol.messages.*;
```

Comment:
Can we not do the `*` import here but individual imports instead?

Reply:
Fixed. Also fixed my Intellij settings

Reply:
We should commit those settings to Git.